src/dataflow/deadlines.rs

```diff
@@ -1,10 +1,22 @@
-use crate::dataflow::{stream::StreamId, time::Timestamp};
+use crate::dataflow::{stream::StreamId, time::Timestamp, StateT};
 use std::{
     collections::{HashMap, HashSet},
     sync::{Arc, Mutex},
 };
 use tokio::time::Duration;
 
+/*************************************************************************************************
+ * DeadlineId: The identifier assigned to each deadline in order to enable handler invocation.   *
+ ************************************************************************************************/
+
+/// The identifier assigned to each deadline installed in a SetupContext.
+pub type DeadlineId = crate::Uuid;
+
+/*************************************************************************************************
+ * CondFn: The type of the start and end condition functions invoked by the different types of   *
+ * deadlines to decide when to initiate and finish the deadline.                                 *
+ ************************************************************************************************/
```

Comment:
Same here, not sure this is necessary since we already have documentation above the `CondFn` trait.